SparkPost/2web2ui
# AC-1970: Global tracking for all forms in Alert, Api-keys and Auth

AC-1970: Global tracking for all forms in Alert, Api-keys and Auth

### What Changed

- Allowlisted forms of Alert, Api-keys and Auth
- Added cypress tests for the above forms


### How To Test

- Check for form submissions on Alert.
- Check for form submissions on Api-keys page.
- Check for form submissions on Auth page and if all the sensitive fields are blocklisted

### To Do
